components/button/button.js

```diff
@@ -0,0 +1,109 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ButtonMixin } from './button-mixin.js';
+import { buttonStyles } from './button-styles.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+import { VisibleOnAncestorMixin } from '../../mixins/visible-on-ancestor-mixin.js';
+
+class Button extends ButtonMixin(VisibleOnAncestorMixin(RtlMixin(LitElement))) {
+
+	static get styles() {
+		return [ buttonStyles,
+			css`
+
+			:host {
+				display: inline-block;
+			}
+			:host([hidden]) {
+				display: none;
+			}
+			button {
+				font-family: inherit;
+				padding: 0.5rem 1.5rem;
+				@apply --d2l-button-shared;
+				@apply --d2l-label-text;

```

Comment:
To use the label text styles, you'll want to use the `d2l-label-text` class on the button (see button-subtle), and will need to use the labelStyles from typography. Also you don't need these `@apply`s, as I believe all the necessary style is in the shared `buttonStyles` css

Reply:
Yeah this is one of the big shifts when moving from Polymer -> Lit. The mixins go away and we use the style imports instead.